frontend/docs/pages/launches/rate-limits.mdx

```diff
@@ -0,0 +1,158 @@
+  <Tabs.Tab>
+1) Define your rate limits in code:
+
+```go
+err = c.Admin().PutRateLimit("example-limit", &types.RateLimitOpts{
+    Max:      3,
+    Duration: "minute",
+})
+```
+
+2. Then, specify the consumption of limits in step definitions:
+
+```go
+err = w.On(
+    worker.NoTrigger(),
+    &worker.WorkflowJob{
+        Name:        "rate-limit-workflow",
+        Description: "This illustrates rate limiting.",
+        Steps: []*worker.WorkflowStep{
+            worker.Fn(StepOne).SetName("step-one").SetRateLimit(
+                worker.RateLimit{
+                    Units: 1,
+                    Key:   "example-limit",
+                },
+            ),
+        },
+    },
+)
+```
+
+</Tabs.Tab>
+</Tabs>
+Rate limits are now supported in our Python, Go, and TypeScript SDKs. We can't wait to see how you use them to keep your workflows running as fast as possible!
+
+[Docs →](https://docs.hatchet.run/home/features/rate-limits)
+
+## Context Streaming
+
+Stream progress and results from your background worker to the frontend
+
+Imagine your customer kicks off a long batch job, then stare at the "In Progress" spinner for hours, wishing they had a clue what was happening behind the scenes... so they leave to take a nap, never to return 😴
+
+Keep them in the loop by streaming arbitrary data (text, intermediate image frames, etc) from within the step run execution!
+
+<Tabs items={['Python', 'Typescript']}>
+  <Tabs.Tab>
+```python
+@hatchet.step()
+def step1(self, context: Context):
+    # Stream some data from the step context
+    context.put_stream('hello from step1') # <-- any arbitrary data under ~4mb!
+    # continue with the step run...
+    return {"step1": "results"}
+    ```
+</Tabs.Tab>
+  <Tabs.Tab>
+```tsx
+async function step1(ctx: Context) {
+    // Stream some data from the step context
+    ctx.putStream('step1 stream'); // <-- any arbitrary data under ~4mb!
+    // continue with the step run...
+    return { step1: 'step1 results!' };
+}
+```
+</Tabs.Tab>
+</Tabs>
+
+[Docs →](https://docs.hatchet.run/home/features/streaming#streaming-from-a-step-context)
+
+## Cancel Running Tasks from the Dashboard
+
+We added a button to send the cancelation signal to your running step!
+
+We've all been there. You're excitedly configuring your workflow, and suddenly your finger slips and you accidentally kick off a massive batch job prematurely. Oops! 😅
+
+Or maybe you've been staring at a step run for what feels like an eternity, and you're pretty sure it's stuck in some sort of infinite loop purgatory.
+
+Now you can cancel a workflow run with a single click:
+
+![Untitled](/cancel-in-prog.gif)
+
+[Docs →](https://www.notion.so/Release-v0-20-e1c2cfb4569d41aaba9f4a4cd8919a7d?pvs=21)
+
+### Smaller Improvements
+
+- Github SSO option on [Hatchet Cloud](https://cloud.onhatchet.run/)
+- [Interactive Tutorial](https://cloud.onhatchet.run/onboarding/get-started) to quickly learn the fundamentals of getting started with Hatchet
```

Comment:
People won't be redirected to this after login - and this exists on self-hosted too, right?

Reply:
yeah i thought about this too, i think just linking is the best case. if they're signed in they will see the tutorial. if they're not signed in they'll need to sign up and then see the tutorial.

also on self-host but i think we focus on cloud here